### PR TITLE
Bump to PSY3, PF 0.6 and PNM 0.9

### PR DESCRIPTION
**Pull request by rodrigomha**

### CODE REVIEW

File: src/utils/psy_utils.jl
Comment (rodrigomha) [inline src/utils/psy_utils.jl:1]: @jd-lara We need to move this to PSY eventually

File: src/utils/psy_utils.jl [hunk 1]
```diff
@@ -148,3 +148,8 @@ function transform_load_to_constant_power(load::PSY.StandardLoad)
     PSY.set_max_impedance_reactive_power!(load, 0.0)
     return
 end
+
+# TO REMOVE AFTER UPDATE IN PSY
+function PSY.get_reactive_power_limits(x::PSY.Source)
```
Comment (rodrigomha) [inline src/utils/psy_utils.jl:153]: @jd-lara we need to move this to PSY eventually

Comment (jd-lara) [reply src/utils/psy_utils.jl:153]: Can you open a PR with this too and make a single group of releases please?

**Comment by rodrigomha** [comment]
Tests are passing locally with main of PSY, but they do not pass with 3.1.1

**Pull request merged**